backend/FwLite/MiniLcm.Tests/Validators/ExampleSentenceValidatorTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿using FluentValidation.TestHelper;
+using MiniLcm.Validators;
+
+namespace MiniLcm.Tests.Validators;
+
+public class ExampleSentenceValidatorTests
+{
+    private readonly ExampleSentenceValidator _validator = new();
+
+    [Fact]
+    public void Succeeds_WhenDeletedAtIsNull()
+    {
+        var example = new ExampleSentence() { Id = Guid.NewGuid(), Sentence = new MultiString(){{"en", "sentence"}}, DeletedAt = null };
+        _validator.TestValidate(example).ShouldNotHaveAnyValidationErrors();
+    }
+
+    [Fact]
+    public void Fails_WhenDeletedAtIsNotNull()
+    {
+        var example = new ExampleSentence() { Id = Guid.NewGuid(), DeletedAt = DateTimeOffset.UtcNow };
+        _validator.TestValidate(example).ShouldHaveValidationErrorFor("DeletedAt");
+    }
+
+    [Theory]
+    [InlineData("Sentence")]
+    [InlineData("Translation")]
```

Comment:
should use the `nameof` syntax here as well

Reply:
Done.